sklearn/linear_model/_glm/glm.py

```diff
@@ -522,7 +527,15 @@ def __init__(
 
     @property
     def family(self):
-        # Make this attribute read-only to avoid mis-uses e.g. in GridSearch.
+        """PoissonRegressor.family always returns 'poisson'.
+
+        This attribute is read-only to avoid mis-uses e.g. in GridSearch.
+
+        Returns
+        -------
+        str
+            Returns a string 'poisson'
+        """
```

Comment:
```suggestion
        """str : Returns the string `'poisson'`."""
        # Make this attribute read-only to avoid mis-uses e.g. in GridSearch.
```